app/ui/browser-modern/views/browser/page/index.jsx

```diff
@@ -204,13 +204,30 @@ class Page extends Component {
 
     this.webview.addEventListener('ipc-message', e => {
```

Comment:
Looks like we could do:

        this.webview.addEventListener('mousedown', e => {

        });

And that fires, but 'click' doesn't for some reason.